### DISCUSSION TITLE
i have 5KB image i want to redraw in 500 bytes data without much reduction of quality of image

### DISCUSSION THREAD

**Discussion by lalwanikamaldev** [SixLabors/ImageSharp#2734] [Q&A]
i have 5KB image i want to redraw in 500 bytes data without much reduction of quality of image  .

**Comment by JimBobSquarePants** [comment]
There are likely no image formats that would allow you to do this while maintaining the original dimensions.

Please make an effort to be more clear as to what you are trying to do and why; it’s impossible to help you with such little information.
